content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -97,50 +89,50 @@ from the table.
 
   For example, suppose you create a Git commit with hyper-parameter changes for
   running a new experiment, and your training CI job creates a new Git commit
-  with the experiment results. You may want to hide the first commit and only
-  display the second commit, which has the new values for the hyper-parameters
-  as well as experiment results (metrics). For this, you can use the string
-  `[skip studio]` in the commit message of the first commit.
-
-- **Hide commits and branches manually:** You can selectively hide commits and
-  branches. This can be useful if there are commits that do not add much value
-  in your project. To hide a commit or branch, click on the 3-dot menu next to
-  the commit or branch name and click on `Hide commit` or `Hide branch`.
+  with the experiment results (metrics and plots). You may want to hide the
+  first commit and only display the second commit, which has the new values for
+  the hyper-parameters as well as experiment results. For this, you can use the
+  string `[skip studio]` in the commit message of the first commit.
```

Comment:
Very minor, but the example scenario here is so specific to the way our CI workflow works that I doubt users understand. It might be simpler to use an example like "suppose you create a Git commit with only code changes unrelated to your model."

Reply:
Good point. But the earlier bullet point (`Iterative Studio auto-hides irrelevant commits`) already covers the scenario that you have mentioned as it says `Iterative Studio identifies commits where metrics, files and hyperparameters did not change and hides them automatically.` So, that example would not be enough.

Because `[skip studio]` was introduced specifically for the scenario where there's one commit with just hyperparam changes and another commit which also includes the results, I think it's ok to use the current example (although I agree it's not very easy to understand).